microservices/ingestservice/requirements.txt

```diff
@@ -1,8 +1,5 @@
 fastapi
 uvicorn
-constants
-dataclasses
-pydantic>=2.5.1
```

Comment:
Ohne den pydantic>=2.5.1 requirement bekomme ich den folgenden error:
 File "/home/jess/dev/recommender-explorer/microservices/ingestservice/dto/recoexplorer_item.py", line 2, in <module>
    from pydantic import BaseModel, field_serializer
ImportError: cannot import name 'field_serializer' from 'pydantic'

Reply:
ich hab jetzt alle versionen gepinnt, wenn du nochmal installierst sollte es hinhauen